packages/v3/src/esp-entity-table.ts

```diff
@@ -125,6 +133,33 @@ export class EntityTable extends LitElement implements RestAction {
         this.requestUpdate();
       }
     });
+
+    window.source?.addEventListener("sorting_group", (e: Event) => {
+      const messageEvent = e as MessageEvent;
+      const data = JSON.parse(messageEvent.data);
+      const groupIndex = this.groups.findIndex((x) => x.name === data.name);
+      if (groupIndex === -1) {
+        let group = {
+           ...data,
+        } as groupConfig;
+        this.groups.push(group);
+        this.groups.sort((a, b) => {
+          return a.sorting_weight < b.sorting_weight  
+            ? -1  
+            : 1  
+        });
+      }
+    });
+
+    this.groups = EntityTable.ENTITY_CATEGORIES.map((category, index) => ({
+      name: category,
+      sorting_weight: index - 1000
+    }));
+
+    this.groups.push({
+      name: EntityTable.ENTITY_UNDEFINED,
+      sorting_weight: -10001 
```

Comment:
I was curious about the magic numbers for `index - 1000` and `sorting_weight: -10001` -- wondering if using a descriptive constant at the top might help?

Reply:
to sure if they are needed at all. I was just tring to make sure that the `entity_category` would be allways displayed first, since the wil get a sorting weight of 0,1 and 2